provider.tf

```diff
@@ -1,3 +1,4 @@
 provider "aws" {
-  region  = "us-east-1"
+  region  = "eu-north-1"
 }
+ 
```

Comment:
could you please, add newline at the end of file.

Reply:
Thank you for the review, done.